diff --git a/datasets/protein_ligand.py b/datasets/protein_ligand.py
index 377e8a8..a952dfc 100644
--- a/datasets/protein_ligand.py
+++ b/datasets/protein_ligand.py
@@ -47,7 +47,7 @@ def get_ligand_atom_features(rdmol):
     hs = (node_type == 1).to(torch.float)
     num_hs = scatter(hs[row], col, dim_size=num_atoms).numpy()
     # need to change ATOM_FEATS accordingly
-    feat_mat = np.array([atomic_number, aromatic, degree, num_hs, hybrid], dtype=np.long).transpose()
+    feat_mat = np.array([atomic_number, aromatic, degree, num_hs, hybrid], dtype=np.int64).transpose()
     return feat_mat
 
 
@@ -93,8 +93,8 @@ def parse_sdf_file_text(path):
         col += [end, start]
         edge_type += 2 * [bond_type_map[int(bond_line[6:9])]]
 
-    edge_index = np.array([row, col], dtype=np.long)
-    edge_type = np.array(edge_type, dtype=np.long)
+    edge_index = np.array([row, col], dtype=np.int64)
+    edge_type = np.array(edge_type, dtype=np.int64)
 
     perm = (edge_index[0] * num_atoms + edge_index[1]).argsort()
     edge_index = edge_index[:, perm]
@@ -163,7 +163,7 @@ def parse_sdf_file_mol(path, heavy_only=True, mol=None):
     # factory = ChemicalFeatures.BuildFeatureFactory(fdefName)
     # rdmol = next(iter(Chem.SDMolSupplier(path, removeHs=heavy_only)))
     # rd_num_atoms = rdmol.GetNumAtoms()
-    # feat_mat = np.zeros([rd_num_atoms, len(ATOM_FAMILIES)], dtype=np.long)
+    # feat_mat = np.zeros([rd_num_atoms, len(ATOM_FAMILIES)], dtype=np.int64)
     # for feat in factory.GetFeaturesForMol(rdmol):
     #     feat_mat[feat.GetAtomIds(), ATOM_FAMILIES_ID[feat.GetFamily()]] = 1
 
@@ -195,8 +195,8 @@ def parse_sdf_file_mol(path, heavy_only=True, mol=None):
         row += [start, end]
         col += [end, start]
         edge_type += 2 * [BOND_TYPES[bond.GetBondType()]]
-    edge_index = np.array([row, col], dtype=np.long)
-    edge_type = np.array(edge_type, dtype=np.long)
+    edge_index = np.array([row, col], dtype=np.int64)
+    edge_type = np.array(edge_type, dtype=np.int64)
     perm = (edge_index[0] * num_atoms + edge_index[1]).argsort()
     edge_index = edge_index[:, perm]
     edge_type = edge_type[perm]
diff --git a/utils/data.py b/utils/data.py
index 71a64d0..6b25a02 100644
--- a/utils/data.py
+++ b/utils/data.py
@@ -150,17 +150,17 @@ class PDBProtein(object):
 
     def to_dict_atom(self):
         return {
-            'element': np.array(self.element, dtype=np.long),
+            'element': np.array(self.element, dtype=np.int64),
             'molecule_name': self.title,
             'pos': np.array(self.pos, dtype=np.float32),
-            'is_backbone': np.array(self.is_backbone, dtype=np.bool),
+            'is_backbone': np.array(self.is_backbone, dtype=bool),
             'atom_name': self.atom_name,
-            'atom_to_aa_type': np.array(self.atom_to_aa_type, dtype=np.long)
+            'atom_to_aa_type': np.array(self.atom_to_aa_type, dtype=np.int64)
         }
 
     def to_dict_residue(self):
         return {
-            'amino_acid': np.array(self.amino_acid, dtype=np.long),
+            'amino_acid': np.array(self.amino_acid, dtype=np.int64),
             'center_of_mass': np.array(self.center_of_mass, dtype=np.float32),
             'pos_CA': np.array(self.pos_CA, dtype=np.float32),
             'pos_C': np.array(self.pos_C, dtype=np.float32),
@@ -264,8 +264,8 @@ def parse_sdf_file(path):
         col += [end, start]
         edge_type += 2 * [BOND_TYPES[bond.GetBondType()]]
 
-    edge_index = np.array([row, col], dtype=np.long)
-    edge_type = np.array(edge_type, dtype=np.long)
+    edge_index = np.array([row, col], dtype=np.int64)
+    edge_type = np.array(edge_type, dtype=np.int64)
 
     perm = (edge_index[0] * rd_num_atoms + edge_index[1]).argsort()
     edge_index = edge_index[:, perm]
